include/IECoreHoudini/OBJ_SceneCacheTransform.h

```diff
@@ -105,7 +105,9 @@ class OBJ_SceneCacheTransform : public OBJ_SceneCacheNode<OBJ_SubNet>
 			UT_String shapeFilter;
 			UT_String tagFilterStr;
 			UT_StringMMPattern tagFilter;
+			UT_String tagGroupsPrefix;
```

Comment:
Rather than expose the tagGroupsPrefix as a parm, maybe we should just hardcode "ieTag_" since thats the only thing the `IECore::HoudiniLiveScene` and `ROP_SceneCacheWriter` will recognize. It seems easy enough to rename the prim groups with a downstream SOP if needed. Or if you're keen to keep the parm, maybe it should default to "ieTag_"?